src/sql/workbench/api/common/sqlExtHostTypes.ts

```diff
@@ -697,9 +697,9 @@ export class ConnectionProfile {
 		this.options['azureTenantId'] = value;
 	}
 
-	options: Map<string, any> = new Map<string, any>();
+	options: { [key: string]: any } = {};
 
-	static createFrom(options: Map<string, any>): ConnectionProfile {
+	static createFrom(options: { [key: string]: any }): ConnectionProfile {
```

Comment:
Found a bug here. Map =/= {}. how this is currently being used it is a json object not a map.